backends/dropbox/index.js

```diff
@@ -37,11 +54,13 @@ export default class Dropbox extends OAuthBackend {
 	 * @param {Object} file - An object with name & data keys
 	 * @return {Promise} A promise that resolves when the file is saved.
 	 */
-	put (serialized, {path = this.path, ...o} = {}) {
+	async put (data, {ref, isUploading} = {}) {
```

Comment:
Does that mean that users now need to provide their path as `{ref: {path: "images"}}`? That's a bit clumsy. I think the previous API made more sense.

Reply:
Good catch! Thank you. Fixed.

Reply:
In general, when designing APIs, it usually pays off to start by writing *calls* to the API (i.e. how would it look like when used), rather than implementing it. In fact, I often recommend people start by writing documentation. 
I think that's actually one of the benefits of TDD: it forces you to *use* your API before you implement it.

Reply:
I will do my best to follow your advice. Thank you!